### DISCUSSION TITLE
Product Attributes

### DISCUSSION THREAD

**Discussion by rmiller335** [getcandy/hub#69] [General]
I have added an attribute group under Product, called Crankset Specs. It contains an attribute named Derailleur Compatibility.  However, when I go to edit my product I don't see any attributes.  What am I missing?

Thanks,
Bob Miller

**Comment by glennjacobs** [comment]
I believe you need to add the attributes to the Product Type you are using, so they display.

**Comment by rmiller335** [comment]
That did it, thanks!

**Reply by glennjacobs** [reply]
Awesome. Now... if you could sort me a Shimano XT m8100 groupset... ;-)

**Comment by rmiller335** [comment]
Good luck with that!
